web/styles/settings.css

```diff
@@ -480,8 +480,14 @@ input[type="checkbox"] {
     align-items: center;
 }
 
-.inline-block.organization-permissions-parent div:first-of-type {
-    margin-top: 10px;
+.inline-block.organization-permissions-parent {
+    & div:first-of-type {
+        margin-top: 10px;
+    }
+
+    .dropdown-list-widget div:first-of-type {
```

Comment:
A better fix here would to make the selector for setting margin-top to 10px more selective probably by using `input-group` class.